sauce/features/budget/check-credit-balances/settings.js

```diff
@@ -1,8 +1,29 @@
 module.exports = {
   name: 'CheckCreditBalances',
-  type: 'checkbox',
+  type: 'select',
   default: false,
   section: 'budget',
   title: 'Paid in Full Credit Card Assist',
-  description: 'Highlights credit card category balances with a yellow warning if the balance of the category does not match the account balance. Adds a button to the Inspector to rectify the difference.'
+  description: 'This keeps track of Credit Card Payments that aren\'t fully budgeted by either highlighting and providing a rectify button or automatically rectifying..',
+  options: [
+    { "name": "Disabled", "value": "disabled" },
+    { "name": "Highlight Mismatch In Yellow and add Rectify Button to the inspector", "value": "highlight" },
+    { "name": "Automatically Rectify", "value": "rectify" },
+    { "name": "Automatically Rectify accounts with '#PIF' in the name only", "value": "rectifyPIF" },
+  ], 
+  actions: {
```

Comment:
The `actions` section is no longer used and should be removed. Also, single tics should be used instead of double tics for literals. The `name` and `value` keywords should not be quoted.